bucket/poetry.json

```diff
@@ -4,22 +4,19 @@
     "homepage": "https://python-poetry.org/",
     "license": "MIT",
     "depends": "python",
-    "url": "https://raw.githubusercontent.com/python-poetry/poetry/1.1.13/get-poetry.py",
-    "hash": "e973b3badb95a916bfe250c22eeb7253130fd87312afa326eb02b8bdcea8f4a7",
+    "url": "https://install.python-poetry.org#/install-poetry.py",
+    "hash": "761cdefb65de97882482d5edf29de6d90e42917447aabfde63a693206c4550c6",
     "installer": {
-        "script": "$env:POETRY_HOME=$dir; python $dir/$fname --version $version --no-modify-path -y"
-    },
-    "uninstaller": {
-        "script": "$env:POETRY_HOME=$dir; python $dir/get-poetry.py --uninstall -y"
-    },
-    "bin": "bin\\poetry.bat",
-    "env_set": {
-        "POETRY_HOME": "$dir"
+        "script": [
+            "$env:POETRY_HOME=$dir",
```

Comment:
```suggestion
            "$env:POETRY_HOME=\"$dir\"",
```

We still need quotes here as $dir can include spaces.

Reply:
![screenshot_20220318160028](https://user-images.githubusercontent.com/4353480/159038564-5f48339e-5e95-40a7-a453-d17c70e63c66.png)